packages/app/src/components/DBRowTable.tsx

```diff
@@ -458,226 +458,235 @@ export const RawLogTable = memo(
     });
 
     return (
-      <div
-        className="overflow-auto h-100 fs-8 bg-inherit"
-        onScroll={e => {
-          fetchMoreOnBottomReached(e.target as HTMLDivElement);
-
-          if (e.target != null) {
-            const { scrollTop } = e.target as HTMLDivElement;
-            onScroll?.(scrollTop);
-          }
-        }}
-        ref={tableContainerRef}
-        // Fixes flickering scroll bar: https://github.com/TanStack/virtual/issues/426#issuecomment-1403438040
-        // style={{ overflowAnchor: 'none' }}
-      >
-        <table
-          className="w-100 bg-inherit"
-          id={tableId}
-          style={{ tableLayout: 'fixed' }}
+      <>
+        {!isLoading && explainResults && (
```

Comment:
looking at the UI a bit more, I wonder if it makes sense being next to `SearchTotalCount` in the DBSearch page as opposed to being part of the table component?

thinking something like

<img width="853" alt="image" src="https://github.com/user-attachments/assets/611e05cb-b3e2-4264-987d-42815712b842" />

Reply:
Awesome, yeah. This is the feedback I was looking for. I like that better. I can move it.